source/install-desktops/enable-reverse-proxy.rst

```diff
@@ -0,0 +1,149 @@
+.. _install-desktops-enable-reverse-proxy:
+
+Enable Reverse Proxy
+====================
+
+The reverse proxy will proxy a request to any specified host and port through
```

Comment:
I think its safe to remove this sentence: 

>This is different than what is used for proxying to per-user NGINX processes through Unix domain sockets. 

Also the words "reverse proxy" could be linked externally to https://en.wikipedia.org/wiki/Reverse_proxy